src/plugins/poap/components/CustomBlock.vue

```diff
@@ -147,7 +147,11 @@ export default {
 </script>
 
 <template>
-  <BaseBlock title="I voted POAP" :loading="loading">
+  <BaseBlock
+    title="I voted POAP"
+    :loading="loading"
+    v-if="currentState !== 'NO_POAP'"
```

Comment:
```suggestion
    v-if="currentState !== 'NO_POAP'"
    title="I voted POAP"
    :loading="loading"
```

`v-if` should come first

Reply:
Also with this change the loading is redundant, as the block will not be visible until loading is complete. I'm not sure this is the best approach

Reply:
What do you think is the best approach then?

We should show loading and then hide?
Or we remove loading completely?

Reply:
Now it displays loading, if POAP doesn't exist we hide the block, else keep it

Reply:
If the Poap plugin is setup for the space we should always show the POAP block and loading on the proposal. And if there is no POAP for the specific proposal then it should show a message inside the block. @ChaituVR

Reply:
But this is the already current functionality 😄 It is confusing that we show a message on proposals that don't have POAP. so we wanted to hide no? 🤔

Reply:
Could you explain your concerns with pitch owners on Discord plz? so they can take a decision

Reply:
> ```suggestion
>     v-if="currentState !== 'NO_POAP'"
>     title="I voted POAP"
>     :loading="loading"
> ```
> 
> `v-if` should come first